rules/falco_rules.yaml

```diff
@@ -2648,7 +2648,7 @@
     command=%proc.cmdline container_id=%container.id container_name=%container.name image=%container.image.repository:%container.image.tag)
   priority:
     NOTICE
-  tags: [process, mitre_persistence]
+  tags: [process, mitre_TA0003_persistence, mitre_TA0004_privilege_escalation, mitre_T1068_explotation_for_privilege_escalation]
```

Comment:
`mitre_T1068_explotation_for_privilege_escalation` probably is not the right fit here.